python/lsst/ts/ATPneumaticsSimulator/pneumatics_csc.py

```diff
@@ -140,16 +140,16 @@ def initialize(self):
         self.set_cell_vents_events(closed=True, opened=False)
         self.set_m1_cover_events(closed=True, opened=False)
         self.evt_instrumentState.set_put(
-            state=ATPneumatics.AirValveState.VALVEOPENED,
+            state=ATPneumatics.AirValveState.OPEN,
```

Comment:
This should be VALVEOPENED like it was, to match the names in the XML. Similarly for CLOSE.

If we wanted to reduce stutter we'd have to fix the XML. But even removing the stutter would leave CLOSED/OPENED, not CLOSE/OPEN. It's a state, not a command.